src/collectors/mesos/mesos.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+
+"""
+
+Collects metrics from a mesos instance (master or slave).
+
+#### Example Configuration
+
+Typically metrics for the master are exposed on port 5050,
+while metrics for the slaves are exposed on port 5051.
+
+```
+    host = localhost
+    port = 5050
+```
+"""
+
+import diamond.collector
+import json
+import urllib2
+
+
+class MesosCollector(diamond.collector.Collector):
+
+    METRICS_PATH = "metrics/snapshot"
+
+    def get_default_config_help(self):
+        config_help = super(MesosCollector,
+                            self).get_default_config_help()
+        config_help.update({
+            'host': 'Hostname',
+            'port': 'Port',
+            'path': 'Metrics Path'
```

Comment:
Just trying to line this up with what's in `get_default_config` down below. Looking at this, though, I don't think I need this here (in `get_default_config_help`). Just `get_default_config`.

Most (if not all?) collectors do provide a path in `get_default_config`. Examples:

* [CPU](https://github.com/python-diamond/Diamond/blob/2c0de81281a6750cf06fac760c081f89a088bca4/src/collectors/cpu/cpu.py#L56)
* [Disk](https://github.com/python-diamond/Diamond/blob/2c0de81281a6750cf06fac760c081f89a088bca4/src/collectors/diskusage/diskusage.py#L60)
* [Network](https://github.com/python-diamond/Diamond/blob/2c0de81281a6750cf06fac760c081f89a088bca4/src/collectors/network/network.py#L42)

The intent is for these to end up being something like `servers.hostname.mesos.master.value`.